registry.k8s.io/images/k8s-staging-dra-driver-cpu/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- dchen1107
+- johnbelamaric
+- klueska
```

Comment:
/cc @klueska @pohly - adding all `wg-device-management-leads` as approvers.

Reply:
We should add the owners of the source repo (that may be the same thing in this particular case)

Reply:
Yes, all the owners from https://github.com/kubernetes-sigs/dra-driver-cpu/blob/main/OWNERS are added as approvers here.